code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1977,7 +1946,11 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 /datum/species/proc/space_move(mob/living/carbon/human/H)
 	if(H.movement_type & FLYING)
 		return TRUE
-	return FALSE
+
+	var/maybe_wings = H.getorganslot(ORGAN_SLOT_EXTERNAL_WINGS)
+	if(istype(maybe_wings, /obj/item/organ/external/wings/moth))
```

Comment:
This typecheck doesn't make sense. Wings in general should just have a proc that says whether they are considered flying or not, or even *better*, just apply a trait for when you are.